<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateProduction</class>
 <widget class="QDialog" name="CreateProduction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create/Update production</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item row="4" column="2">
    <widget class="QLineEdit" name="prodCountEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="expiryDateLb">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Expiry date:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="sessionStartLb">
     <property name="text">
      <string>Session start time:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="SessionEndLb">
     <property name="text">
      <string>Session end time:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="addProdBtn">
     <property name="text">
      <string>Add products</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="productionDateLb">
     <property name="text">
      <string>Production date:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="prodCountLb">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Product count:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QTimeEdit" name="sesEndTimeEdit"/>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QTimeEdit" name="sesStartTimeEdit"/>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QDateEdit" name="prdDateEdit"/>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QDateEdit" name="expiryDateEdit"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sesStartTimeEdit</tabstop>
  <tabstop>sesEndTimeEdit</tabstop>
  <tabstop>addProdBtn</tabstop>
  <tabstop>prodCountEdit</tabstop>
  <tabstop>okBtn</tabstop>
  <tabstop>cancelBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
